### ISSUE TITLE
Model evaluator development

### ISSUE THREAD

**Issue by PeterCalifano** [PeterCalifano/pyTorchAutoForge#10]
Functionalities to add:

- [x] 1. Single model evaluation using custom loss function and metrics with statistics and plots
- [ ] 2. Multiple model ("batch") evaluation with [1] and comparison plots
